content/millennium/r4/encounters/encounter.md

```diff
@@ -10,3 +10,90 @@ title: Encounter | R4 API
 ## Overview
 
 Placeholder
+
+## Search
+
+Search for Encounters that meet supplied query parameters:
+
+    GET /Encounter?:parameters
+
+_Implementation Notes_
+
+* The [Encounter.hospitalization.destination] will be returned as a reference to a [contained] location resource.
+* The [Encounter.location.location] may be returned as a reference to a [contained] location resource.
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true)%>
+
+### Parameters
+
+ Name       | Required?                  | Type          | Description
+------------|----------------------------|---------------|-------------------------------------------------------------------------------------------------------
+ `_id`      | This or patient or subject | [`token`]     | The logical resource id associated with the Encounter. Example: `7891`
+ `patient`  | This or subject or _id     | [`reference`] | The patient present at the encounter. Example: `12345`
+ `subject`  | This or patient or _id     | [`reference`] | The patient present at the encounter. Example: subject=Patient/1316024 or subject:Patient=1316024
```

Comment:
subject examples need some tildas

Reply:
tildes added to description and required? sections